package.json

```diff
@@ -13,62 +13,39 @@
         "access": "public"
     },
     "scripts": {
-        "extract-pot": "d2-i18n-extract -p src/ -o i18n/",
-        "generate-locale": "d2-i18n-generate -n 'ui-forms' -p ./i18n/ -o ./src/locales/",
-        "localize": "yarn extract-pot && yarn generate-locale",
-        "prebuild": "rm -rf ./build/* && yarn localize",
-        "build:commonjs": "BABEL_ENV=commonjs babel src --out-dir ./build/cjs --copy-files --verbose",
-        "build:modules": "BABEL_ENV=modules babel src --out-dir ./build/es --copy-files --verbose",
-        "build": "NODE_ENV=production yarn build:commonjs && NODE_ENV=production yarn build:modules",
-        "postbuild": "NODE_ENV=production yarn build-storybook",
-        "build-storybook": "build-storybook -c .storybook -o ./build/docs",
-        "start-storybook": "start-storybook --port 5000",
-        "prestart": "yarn localize",
-        "start": "yarn start-storybook",
-        "start:testing": "STORYBOOK_INCLUDE_TESTING=1 yarn start --ci --port $PORT",
-        "cypress:browser": "cypress open --config baseUrl=http://localhost:$PORT",
-        "cypress:ci": "NO_COLOR=1 cypress run -b $BROWSER --config baseUrl=http://localhost:$PORT",
-        "cypress:open": "PORT=12345 concurrently --kill-others -n storybook,cypress 'yarn start:testing' 'wait-on http-get://localhost:$PORT && yarn cypress:browser'",
-        "cypress:run": "PORT=12345 concurrently --kill-others --success first -n storybook,cypress 'yarn start:testing' 'wait-on http-get://localhost:$PORT && yarn cypress:ci'",
-        "test": "jest"
-    },
-    "husky": {
-        "hooks": {
-            "commit-msg": "d2-style commit check",
-            "pre-commit": "yarn test && d2-style js apply"
-        }
-    },
-    "cypress-cucumber-preprocessor": {
-        "nonGlobalStepDefinitions": true
+        "prebuild": "rm -rf ./build/*",
```

Comment:
This isn't necessary, it's [done automatically](https://github.com/dhis2/app-platform/blob/master/cli/src/commands/build.js#L72) by `d2-app-scripts build`

Reply:
Nice, good to know. Will remove here and in ui-core and ui-widgets.